code/modules/virus2/effect/stage_3.dm

```diff
@@ -190,6 +191,30 @@
 		mob.equip_to_slot(virusclown_hat, slot_wear_mask)
 	mob.reagents.add_reagent(PSILOCYBIN, 20)
 	mob.say(pick("HONK!", "Honk!", "Honk.", "Honk?", "Honk!!", "Honk?!", "Honk..."))
+	if(ishuman(mob))
+		var/mob/living/carbon/human/affected = mob
+		if(multiplier >=2) //clown shoes added
+			var/obj/item/clothing/shoes/clown_shoes/virusshoes = new /obj/item/clothing/shoes/clown_shoes
```

Comment:
seems to me that this creates shoes in nullspace and only equips them if necessary, which means when this activates again to do the suit and dna, it's going to keep creating null shoes

Reply:
virus multipliers don't increase or decrease inside the body without badmin intervention / varediting, so not sure this would actually be the case

Reply:
this is going to happen every activate() anyway, you're creating the shoes up here above the check to see if they are wearing the shoes